new-project/src/components/SandboxComponent/index.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+
+const User = () => {
+  const [name, setName] = useState("Tom");
+
+  useEffect(() => {
+    document.title = `Привет ${name}`;
```

Comment:
This code will be triggered each time when this component will be updated, need to add an array of dependencies